src/django/api/views.py

```diff
@@ -336,12 +336,13 @@ def create(self, request):
             replaces.save()
 
         for idx, line in enumerate(csv_file):
-            new_item = FacilityListItem(
-                row_index=idx,
-                facility_list=new_list,
-                raw_data=line.decode().rstrip()
-            )
-            new_item.save()
+            if idx > 0:
+                new_item = FacilityListItem(
+                    row_index=(idx - 1),
```

Comment:
Can we start this index at 1 now? The row index is propagated through the UI and in CSVs that users can download and the intention is for them to be able to collate the process list item row indexes with the row indexes in the uploaded CSVs.

Reply:
Saw Justin's comment on #241 and now I think we might just want to make a new serializer method field which returns a different value for the Row Index to the UI which would correspond to spreadsheet row numbering.